rust/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+#![doc = include_str!("../README.md")]
```

Comment:
(comment) This kind of thing is cool, and I didn't know worked

Reply:
Only found out about this recently, I like it too :-)
This also makes sure the example is executed in `cargo test`.